test/Client.test.ts

```diff
@@ -1,5 +0,0 @@
-/* TODO: real unit tests */
```

Comment:
Why delete this file (and `Daemon.test.ts` too)?

Reply:
Good question 😂 I think I didn't intend to delete it but I renamed them a few times and and they got lost; I'll add it back. (Though not with actual testcases yet, I want to add tests only once we switch away from this HTTP demo to our actual protocol).

Reply:
Actually, Jest doesn't let you have a suite without testcases, and the current dummy testcase isn't valid (since the code it "tested" is gone). Rather than create yet another dummy testcase I'd rather delete it; we definitely will be adding unit tests to all files in the future.